### ISSUE TITLE
Input Time Picker - Ensure all interactive functionality is operable with the keyboard - (2036570010)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-components#5584]
### Module: 
 17b Input Time Picker 

### Violation: 
 Ensure all interactive functionality is operable with the keyboard 

### WCAG Reference: 
- [2.1.1 Keyboard](https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html) 

### Instance ID: 
 2036570010 

### Severity: 
 8 

### Description: 
 [Issue]
There are interactive controls that cannot be navigated to and/or operated with the keyboard alone. Examples include:

- Increase/decrease hour button
- Increase/decrease minute button
- Increase/decrease second button

[User Impact]
Keyboard-only users will be prevented from accessing the functionality provided by these controls.

[Code Reference] 
 ```
<span aria-label="Increase hour" class="button button--hour-up button--top-left" role="button" tabindex="-1">
<calcite-icon aria-hidden="true" icon="chevron-up" scale="s" calcite-hydrated=""></calcite-icon>
</span>
 ``` 

### Note: 
 [Suggestion]
Ensure all interactive functionality is operable with the keyboard. The best way to accomplish this is by using appropriate native controls, which come with keyboard functionality built in.

In this case it is recommended to remove tabindex="-1" from the instance.

[Compliant Code Example] 
 ```
<span aria-label="Increase hour" class="button button--hour-up button--top-left" role="button">
<calcite-icon aria-hidden="true" icon="chevron-up" scale="s" calcite-hydrated=""></calcite-icon>
</span>
 ```
 

### Media Type: 
 Keyboard Accessibility 

### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.
